extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java

```diff
@@ -97,6 +98,35 @@ public static void addProject(Language language, ProjectType project, List<Depen
 		write(wr, "}");
 	}
 
+	public static void addAssetIndexTask(String assetsDir, BufferedWriter wr) throws IOException {
+		 /**
```

Comment:
Please use the libgdx formatter on any and also only lines you change. Also please follow proper javadoc guidelines for documenting methods.